src/content/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/proactive-detection-applied-intelligence.mdx

```diff
@@ -544,6 +544,32 @@ Proactive Detection sends the event body in JSON format via HTTPS POST. The syst
         Example: v1
       </td>
     </tr>
+    
+    <tr>
+      <td>
+        `viewChartImageUrl`  
+        _string_
+      </td>
+
+      <td>
+        Image showing a chart of the anomalous data.
+
+        [See an example](https://www.example.com/image/8353cf2c-945c-48e8-99de-e903f033a881?height=200&width=400&show_timezone=true).
```

Comment:
@daffinito - this url and the following one are not working for me. Could you check them out, please? Thanks!